stackit/internal/services/iaas/securitygrouprule/resource.go

```diff
@@ -369,6 +371,9 @@ func (r *securityGroupRuleResource) Schema(_ context.Context, _ resource.SchemaR
 							stringvalidator.AtLeastOneOf(
 								path.MatchRoot("protocol").AtName("number"),
 							),
+							stringvalidator.ConflictsWith(
```

Comment:
do we need this? Cant we declare name and number at the same time?

Reply:
Not anymore, IaaS changed the API to be a `oneOf`. Check https://docs.api.eu01.stackit.cloud/documentation/iaas/version/v1beta1#tag/Security-Group/operation/v1beta1CreateSecurityGroupRule